server/src/reporter.ts

```diff
@@ -24,6 +74,56 @@ const validationMessagesEs: ValidationMessage = {
   'nameShouldNotBeKeyword': 'El nombre {0} es una palabra reservada, debe cambiarla',
   'shouldNotBeEmpty': 'El elemento no puede estar vacío: falta escribir código.',
   'shouldUseConditionalExpression': 'Estás usando incorrectamente el if. Devolvé simplemente la expresión booleana.',
+  'shouldPassValuesToAllAttributes' : 'No se encuentra la referencia {0} en {1}',
+  'namedArgumentShouldExist' : 'No se encuentra la referencia {0} en {1}',
+  'linearizationShouldNotRepeatNamedArguments' : 'La referencia {0} está inicializada m\u00E1s de una vez',
+  'namedArgumentShouldNotAppearMoreThanOnce' : 'La referencia {0} está inicializada m\u00E1s de una vez',
+  'shouldInitializeAllAttributes' : 'Debe proveer un valor inicial a las siguientes referencias: {0}',
+  'shouldImplementAllMethodsInHierarchy' : 'Jerarqu\u00EDa inconsistente. Existen m\u00E9todos en mixins que requieren implementaci\u00F3n en super',
+  'shouldUseSelfAndNotSingletonReference' : 'No debe usar el nombre del objeto dentro del mismo. Use \'self\'.',
+  'shouldNotReassignConst' : 'No se pueden modificar las referencias constantes',
+  'shouldNotHaveLoopInHierarchy' : 'La jerarqu\u00EDa de clases produce un ciclo infinito',
+  'shouldOnlyInheritFromMixin' : 'Los mixines solo pueden heredar de otros mixines',
+  'shouldNotDefineMoreThanOneSuperclass' : 'Linearizaci\u00F3n: no se puede definir m\u00E1s de una superclase',
+  'superclassShouldBeLastInLinearization' : 'Linearizaci\u00F3n: la superclase deber\u00EDa estar \u00FAltima en linearizaci\u00F3n',
+  'shouldNotUseOverride' : 'Este m\u00E9todo no sobrescribe ning\u00FAn m\u00E9todo',
+  'possiblyReturningBlock' : 'Este m\u00E9todo devuelve un bloque, si no es la intenci\u00F3n elimine el \'=\' antes de las llaves.',
+  'shouldUseOverrideKeyword' : 'Deber\u00EDa marcarse el m\u00E9todo con \'override\', ya que sobrescribe el de sus superclases',
+  'shouldMatchSuperclassReturnValue' : 'Debe retornar un valor ya que el m\u00E9todo sobrescrito retorna un valor',
+  'getterMethodShouldReturnAValue' : 'Getter debe retornar un valor',
+  'methodShouldHaveDifferentSignature' : 'M\u00E9todo duplicado',
+  'shouldNotDuplicateVariables' : 'Ya existe una variable con este nombre en la jerarqu\u00EDa',
+  'shouldNotDuplicateFields' : 'Ya existe un atributo con este nombre en la jerarqu\u00EDa',
+  'shouldNotDuplicateLocalVariables' : 'Ya existe una variable con este nombre en la jerarqu\u00EDa',
+  'shouldNotDuplicateGlobalDefinitions' : 'Ya existe una definicion con este nombre en la jerarqu\u00EDa',
+  'shouldNotDuplicateVariablesInLinearization' : 'En la jerarqu\u00EDa hay atributos con el mismo nombre: [{0}]',
+  'shouldNotDuplicateEntities' : 'Este nombre ya está definido (importado de {0})',
```

Comment:
```suggestion
  'shouldNotDuplicateEntities' : 'Este nombre ya est\u00E1 definido (importado de {0})',
```

en Eclipse se ve que había bardo, acá quizás no haga falta las tildes